Sofa/Component/src/sofa/component/init.cpp

```diff
@@ -68,6 +72,11 @@ const char* getModuleVersion()
     return MODULE_VERSION;
 }
 
+void registerObjects(sofa::core::ObjectFactory* factory)
+{
+    factory->registerObjectsFromPlugin("Sofa.Component.StateContainer");
```

Comment:
I am annoyed that a meta-module must call the "dynamic" functions (using strings) as if it does not have access to it statically.

Reply:
```cpp
sofa::component::statecontainer::target = "Sofa.Component.StateContainer"
```